exercises/practice/hello-world/lib/hello_world.ex

```diff
@@ -2,8 +2,7 @@ defmodule HelloWorld do
   @doc """
   Simply returns "Hello, World!"
   """
-  @spec hello :: String.t()
```

Comment:
I used the example implementation as the basis for this change and that did no include the `@spec` part. I'm trying to make this file as simple as possible, so I've omitted it. Is that okay?

Reply:
No, we should add it to the example implementation too. All practice exercises stubs have `@spec`s and `@doc`s.

Unfortunately our example implementations get out of date with the stub with regard to documentation. The example says:
```
  Greets the user by name, or by saying "Hello, World!"
  if no name is given
```

Which I guess used to be the case a while ago.

Reply:
I've updated